test/integration/clientset_test.go

```diff
@@ -568,17 +770,23 @@ func testClusterServiceClassClient(sType server.StorageType, client servicecatal
 	return nil
 }
 
-// TestClusterServicePlanClient exercises the ClusterServicePlan client.
-func TestClusterServicePlanClient(t *testing.T) {
+// TestClusterServiceClassClient exercises the ClusterServiceClass client.
+func TestNamespacedServiceClassClient(t *testing.T) {
 	rootTestFunc := func(sType server.StorageType) func(t *testing.T) {
 		return func(t *testing.T) {
-			const name = "test-serviceplan"
+			const name = "test-serviceclass"
+			const namespace = "test-namespace"
+			resetFeaturesFunc, err := enableNamespacedResources()
+			if err != nil {
+				t.Fatal(err)
+			}
+			defer resetFeaturesFunc()
 			client, _, shutdownServer := getFreshApiserverAndClient(t, sType.String(), func() runtime.Object {
-				return &servicecatalog.ClusterServicePlan{}
+				return &servicecatalog.ClusterServiceClass{}
 			})
 			defer shutdownServer()
 
-			if err := testClusterServicePlanClient(sType, client, name); err != nil {
+			if err := testClusterServiceClassClient(sType, client, name); err != nil {
```

Comment:
Let's get the comment, the function name and this call to all align. 😀

Reply:
Done - sorry!